### ISSUE TITLE
EditorTextureManager is never used

### ISSUE THREAD

**Issue by nikoliazekter** [UnderwaterApps/overlap2d#287]
What's the point?

**Comment by azakhary** [comment]
I think it's something from legacy codes. Is it totally never used? Like ever? meaning we can safely  delete it?

**Comment by nikoliazekter** [comment]
Yeah, all instances of EditorTextureManager are never used. And all it does is creating a texture that contains one white pixel.
